pkgs/tools/networking/brook/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "brook";
+  version = "20190401";
+
+  goPackagePath = "github.com/txthinking/brook";
+
+  src = fetchFromGitHub {
+    owner = "txthinking";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "0kx0dpvr3llpdzmw5bvzhdvwkmzrv6kqbsilx6rgrvyl61y9pyry";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/txthinking/brook;
+    description = "A cross-platform Proxy/VPN software";
+    license = with licenses; [ gpl3 ];
+    platforms = with platforms; [ linux ];
```

Comment:
The brackets here are incorrect because `platforms.linux` itself is a list already.  That's why ofborg didn't build anything.
The correct value is `platforms.linux` as suggested earlier.

Reply:
Thanks for the review!
I am sorry for writing `platforms = with platforms; [ linux ];`.